src/stories/Library/material-search/MaterialSearchPreview.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState, useEffect } from "react";
+import Cover from "../cover/Cover";
+import { PreviewData } from "./MaterialSearchExampleData";
+import MaterialSearchLoading from "./MaterialSearchLoading"; // Assuming this is the correct import
+
+interface MaterialSearchPreviewProps {
+  displayMaterial: boolean;
+}
+
+const MaterialSearchPreview = ({
+  displayMaterial,
+}: MaterialSearchPreviewProps) => {
+  const [fictiveLoading, setFictiveLoading] = useState(false);
+
+  useEffect(() => {
+    if (displayMaterial) {
+      setFictiveLoading(true);
+      const timer = setTimeout(() => {
+        setFictiveLoading(false);
+      }, 1500);
+
+      return () => clearTimeout(timer);
+    }
+
+    return () => {};
+  }, [displayMaterial]);
+
+  if (fictiveLoading) {
+    return (
+      <div className="material-search__preview">
+        <div className="material-search__preview-loading">
+          <MaterialSearchLoading loadingText="Loading..." />
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="material-search__preview">
+      {displayMaterial ? (
+        <div className="material-search__preview-material">
+          <Cover src={PreviewData.coverUrl} animate size="large" />
+          <div className="material-search__preview-content">
+            <span className="material-search__preview-content__label">
+              Title:
+            </span>
+            <span>{PreviewData.title}</span>
+            <span className="material-search__preview-content__label">
+              Author:
+            </span>
```

Comment:
Consider reworking this to a [HTML description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) instead of generic spans.

Reply:
yeah thats better 👍